keps/prod-readiness/sig-cluster-lifecycle/2915.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2915
+alpha:
+  approver: "@johnbelamaric"
```

Comment:
According to the spreadsheet this is me and I don't have a ton of reviews so I'll do it... can you update?

Reply:
given @johnbelamaric already approved how about we update the spreadsheet for this particular KEP?

Reply:
Do you have a link to that? This is the first I've seen of this KEP, it appears to be net new